config/v1/types_cluster_version.go

```diff
@@ -406,17 +432,34 @@ type ComponentOverride struct {
 type URL string
 
 // Update represents an administrator update request.
+// +kubebuilder:validation:XValidation:rule="has(self.architecture) && has(self.image) ? (size(self.architecture) == 0 || size(self.image) == 0) : true",message="cannot set both Architecture and Image"
+// +kubebuilder:validation:XValidation:rule="has(self.architecture) && size(self.architecture) != 0 ? size(self.version) != 0 : true",message="Version must be set if Architecture is set"
 // +k8s:deepcopy-gen=true
 type Update struct {
-	// version is a semantic versioning identifying the update version. When this
-	// field is part of spec, version is optional if image is specified.
+	// architecture is an optional field that indicates the desired
+	// value of the cluster architecture. In this context cluster
+	// architecture means either a single architecture or a multi
+	// architecture. architecture can only be set to multi thereby
+	// only allowing updates from single to multi architecture. If
+	// architecture is set image cannot be set and version must be
+	// set. The empty value indicates that architecture is not set
+	// in which case architecture defaults to the current target
+	// release's architecture.
+	//
+	// +optional
+	Architecture ClusterVersionArchitecture `json:"architecture"`
+
+	// version is a semantic versioning identifying the update version.
+	// version is optional if image is specified and required if
+	// architecture is specified.
 	//
 	// +optional
 	Version string `json:"version"`
 
-	// image is a container image location that contains the update. When this
-	// field is part of spec, image is optional if version is specified and the
-	// availableUpdates field contains a matching version.
+	// image is a container image location that contains the update.
+	// image is optional if version is specified and an update exists
+	// with that version in the availableUpdates or history. image
+	// cannot be specified if architecture is specified.
```

Comment:
```suggestion
	// image is a container image location that contains the update.
	// image should be used when the desired version does not exist in availableUpdates or history.
	// When image is set, version is ignored.  When image is set, version should be empty.
	// When image is set, architecture cannot be specified.
```

Reply:
>When image is set, version is ignored.  When image is set, version should be empty.

So this just screams, then why in the heck are "you" forcing me to set it?
>`...message="Version must be set if Architecture is set"`

Reply:
done